test/CI-blackbox/renderer.js

```diff
@@ -0,0 +1,52 @@
+
+const {JavaGenerator, JavaScriptGenerator, TypeScriptGenerator} = require('../../lib');
+const fs = require('fs');
+const path = require('path');
+const languagesToInclude = process.argv.slice(2);
+/**
+ * Processes the input and renders the language the models
+ */
+async function processInput() {
+  const input = JSON.parse(fs.readFileSync(path.resolve(__dirname, './test.json')));
```

Comment:
If you put `input` inside function, you can also put `languagesToInclude` or move `input` outside the function.

Reply:
Gonna keep everything inside `processInput` 🙂